### ISSUE TITLE
YouTube Music feature should be explicitly indicated [Not exactly an issue]

### ISSUE THREAD

**Issue by 19007361** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#3841]
One of the most useful apps I have ever used. According to[ news reports](https://youtube.googleblog.com/2020/05/youtube-music-transfer-google-play-music-library.html?m=1), YouTube Music will soon take over from Google Play Music (before the end of 2020), whereafter GPM will reportedly be discontinued. Of course, this may be extended due to the current pandemic, but eventually, Google seems to aim for YouTube Music as their sole music streaming platform.

Unless I am mistaken, nowhere on this repo or on GPMDP's website does it state anything about actually having a functioning YTM feature, with which I have had no problems thus far. I propose immediately pointing this out, so as to encourage more users to install this. There is no reason for this feature to go unnoticed, and it is impossible to know that this app has YTM built in without installing it first. 

**Checklist:**

* [ x ] I have checked that there are no issues with similar or the same content

  **YOU SHOULD CHECK CLOSED ISSUES AS WELL**
* [ x ] I have checked the FAQ (https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/wiki/FAQ) and the answer I am looking for is not there
* [ x ] I have double checked and can reproduce the issue

**Comment by kudlav** [comment]
Maybe it's even time to swap UI and docs to _Youtube Music main feature_ and present Google Play Music as additional feature.
I'm using this app for Youtube Music since I've installed it and without any problem.

**Comment by 19007361** [comment]
Yeah. I think we should immediately just put YouTube Music on the README and on the website so that it can be indexed in Google searches at least.

**Comment by juanvisbalco** [comment]
I almost skipped the app because I did not know about YT Music support — thankfully I knew to check GitHub.

I would recommend re-branding and refocusing on YT Music rather than GP Music.

**Comment by st3iny** [comment]
This will become even more relevant in the near future due to Google shutting down Play Music later this year.

**Comment by OlJohnny** [comment]
*bump*
This seems even more important than ever, as Google seems to want to now push as many people from GPM to YTM with the new import-your-GPM-library-to-YTM-tool, which is advertised on the YTM start page as of a few days ago.

**Comment by 5jubsDelta** [comment]
Today I've received an E-mail, from google, telling about the discontinuing of GPM, starting from October. So, it's better to do the Rebranding or focus on YTM improvements.

**Comment by aeantipov** [comment]
@19007361 perhaps it might be a good time to revise the name of the issue. Google Music is closing, so there's no point in having it in the name.

**Comment by shaneh445** [comment]
Agreed with all said above. Also wondering if this app will be updated the reflect GPM shutting down

**Comment by andrewjmetzger** [comment]
Very curious to see what happens here. I'm of a similar mind to kudlav on this. Since GPMDP has unique features like skipping disliked songs, last.fm scrobbling, etc., it would be ideal to move these to YTM rather than be forced to contend with the default web interface.

If that's not a realistic goal, so be it. Still, it would be a shame to lose this excellent music player after GPM is gone.

**Comment by 19007361** [comment]
The issue is not that we're gonna lose this music player it's that it already functions for the replacement of Google Music with Youtube Music but nobody knows that until they install it because this repo doesn't mention it anywhere.

**Comment by andrewjmetzger** [comment]
Yeah. Without a vast branding overhaul, I think GPMDP might fall by the wayside as we see alternatives like YTM Desktop come to the fore.

**Comment by MarshallOfSound** [comment]
It's possible that actually just sending people to ytmdesktop might be the best course of action here.  It's either that or we sit down, gather interested parties and figure out how to make GPMDP -> YTM rebrand happen 😆   I'm going to reach out to the YTM desktop folks and see if a discussion there might be productive

**Comment by aeantipov** [comment]
Actually after @andrewjmetzger reply I went and checked out YTM Desktop (which I totally missed) and the experience is largely similar. 
There is some difference in feature support, like "thumbs up forwarding to last.fm" (and I'm sure some more). If there's some merge strategy coming out, one can open a bunch of issues and PRs to translate features.

**Comment by andrewjmetzger** [comment]
I love GPMDP, and have been using it for ages. I would like to see the project stick around, but sending people over to YTM Desktop might not be a bad idea. The lead dev has been pretty responsive in merging those littler "value add"
 features like last.fm integration, lyrics, discord rich presence, etc. that have made GPMDP so great thus far.

If we can't figure out a practical way to transition GPMDP, collaboration sounds like a good place to go. +1

**Comment by andrewjmetzger** [comment]
@aeantipov I only recently discovered YTM Desktop myself as a consequence of the above-mentioned issue #3878. The project seems to be making good progress, and I hope such a positive trend will continue. 

Like you said, I'd like to see as many GPMDP extras make it into YTM Desktop as possible, and I'll be submitting feature requests wherever possible. If nothing else, feature parity would hedge us against the (potential) sunset of GPMDP, and users would be able to choose between two desktop clients.

**Comment by OlJohnny** [comment]
Thanks for mentioning YTM Desktop!
As it seems like development on GPMDP has slowed to a halt, i will probably be moving to YTM Desktop.
As @andrewjmetzger has mentioned, the development of YTM Desktop seems very active and the current stage is really nice to use with a lot of the features we came to like from GPMDP!

**Comment by andrewjmetzger** [comment]
You're welcome; I'm happy to help. Pro-tip if you want the fun features, pull [the `development` branch](https://github.com/ytmdesktop/ytmdesktop/tree/development), and don't use the outdated repo URL in the project README. Hopefully we can get all of the fun stuff in there quickly.

As for GPMDP, if development should ever resume, I'd like nothing more than to support this awesome project. Until then, though: YTM Desktop to the rescue!
